util/reggen/gen_cfg_html.py

```diff
@@ -85,6 +85,46 @@ def gen_cfg_html(cfgs: IpBlock, outfile: TextIO) -> None:
     else:
         genout(outfile, "<p><i>Peripheral Pins for Chip IO: none</i></p>\n")
 
+    # Inter-Module Signals
+    if not cfgs.inter_signals:
```

Comment:
this looks great!
do you think it's worth adding a brief description what the fields mean? I guess specifically for `type` and `act`

Reply:
We have that in the https://docs.opentitan.org/doc/rm/comportability_specification/#inter-signal-handling doc. Do you think it would be helpful to have a link to that section?

Reply:
ah no then i think it's fine.
Do you think it makes sense to link to that in the section? or is that too much?

Reply:
Hum, I thought I pushed before I left the office. Apparently not.. :) I just pushed for the link